apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -17,43 +18,30 @@ export const MultilingualTextPresenter = ({
 }: MultilingualTextPresenterProps): JSX.Element => {
     const { defaultLanguageCode } = useContext(ConfigurableContentContext);
 
-    const { items } = text;
+    const textItemWithDefaultLanguage = getTextItemByLanguageCode(text, defaultLanguageCode);
 
-    const textItemWithDefaultLanguage =
-        items?.find((item) => item.languageCode === defaultLanguageCode) || null;
+    const primaryMultilingualTextItem = isNullOrUndefined(textItemWithDefaultLanguage)
+        ? getOriginalTextItem(text)
+        : textItemWithDefaultLanguage;
 
-    const translations = items?.filter((items) => items.languageCode !== defaultLanguageCode);
+    const { languageCode: languageCodeOfPrimaryTextItem } = primaryMultilingualTextItem;
+
+    const translations = getTextItemsNotInLanguage(text, languageCodeOfPrimaryTextItem);
+
+    const isTranslated: boolean = translations.length > 0 ? true : false;
 
     return (
         <Box width={'fit-content'} data-testid="multilingual-text-display">
-            <Accordion elevation={0}>
-                <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-                    <Typography variant="h4" margin={'auto 0'}>
-                        {isNullOrUndefined(textItemWithDefaultLanguage)
-                            ? 'Translations'
-                            : textItemWithDefaultLanguage.text}
-                        {isNullOrUndefined(textItemWithDefaultLanguage) ? null : (
-                            <Tooltip
-                                title={`${getLabelForLanguage(
-                                    textItemWithDefaultLanguage.languageCode
-                                )}, '${textItemWithDefaultLanguage.role}'`}
-                            >
-                                <IconButton>
-                                    <LanguageIcon />
-                                </IconButton>
-                            </Tooltip>
-                        )}
-                    </Typography>
-                </AccordionSummary>
-                {translations.map(({ languageCode, text, role }) => (
-                    <TranslatedLanguageTextPresenter
-                        key={`${languageCode}-${role}`}
-                        languageCode={languageCode}
-                        text={text}
-                        role={role}
-                    />
-                ))}
-            </Accordion>
+            {isTranslated ? (
```

Comment:
I realized here that there's really two different display states, one with the accordion, one without if there are no translations.  It's awkward to work with the accordion when there are no sub-items for the `AccordionDetails`.